src/PrestaShopBundle/Resources/views/Admin/Module/manage.html.twig

```diff
@@ -39,13 +39,13 @@
       {% include 'PrestaShopBundle:Admin/Module/Includes:menu_top.html.twig' with { 'topMenuData': topMenuData, 'bulkActions': bulkActions } %}
 
       <div id="module-recently-used-list" class="module-short-list">
-        <span id="recently-used_modules" class="module-search-result-wording">{{ 'Recently Used' | trans({}, 'Admin.Modules.Feature') }}</span>
+        <span id="recently-used_modules" class="module-search-result-title">{{ 'Recently Used' | trans({}, 'Admin.Modules.Feature') }}</span>
```

Comment:
Please no useless upper case: _Recently used_. Domains are fine otherwise.

Reply:
Was already defined like that in another PR :/